README.md

```diff
@@ -81,6 +81,19 @@ If you receive a "No machine capacity available" error message when submitting a
 
 FGMachine automatically reloads the `projects.json` file when it is changed.
 
+#### GPU capacity support
+
+In order to handle projects, which require GPUs to perform a task, you need to add two parameters for each project in `projects.json` file:
+
+```json
+{
+  "gpu_capacity": "<gpu capacity needed (as a fraction of one GPU capacity (e.g. 0.5)>",
+  "gpu_command": "<option to pass to script to identify card number (e.g. -gpu)>",
+}
+```
+
+Note that gpu_capacity represents (the inverse of) instances of the program the FGMachine host system can run on one GPU; for example a machine with 4 GPUs, will be able to run 8 instances of the program with capacity 0.1 and gpu_capacity 0.5.
```

Comment:
I think an extra clarifying example would be helpful - you could add "However, if the capacity was 0.25 in the previous example, the machine would only be able to run 4 instances of the program."